hub2labhook/pipeline.py

```diff
@@ -200,7 +214,7 @@ def trigger_pipeline(self):
                           (gevent.head_sha, gevent.commit_url))
             gitbin.push("target", 'HEAD:%s' % gevent.target_refname, "-f")
```

Comment:
Before pushing the whole repo can we very call gitlab to validate that the file a valid .gitlab-ci.yaml  ? 
Code available here:
https://github.com/failfast-ci/ffctl/blob/master/ffctl/commands/lint.py#L5-L14

Reply:
Will be done on a separate PR